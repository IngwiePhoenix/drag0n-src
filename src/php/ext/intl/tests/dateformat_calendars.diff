001+ string(47) "Sunday, January 1, 2012 at 5:12:00 AM GMT+05:12"
002+ string(47) "Sunday, January 1, 2012 at 5:12:00 AM GMT+05:12"
003+ string(48) "Sunday, Tevet 6, 5772 AM at 5:12:00 AM GMT+05:12"
001- string(44) "Sunday, January 1, 2012 5:12:00 AM GMT+05:12"
002- string(44) "Sunday, January 1, 2012 5:12:00 AM GMT+05:12"
003- string(42) "Sunday, Tevet 6, 5772 5:12:00 AM GMT+05:12"