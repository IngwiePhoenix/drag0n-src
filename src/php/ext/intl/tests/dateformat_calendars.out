string(47) "Sunday, January 1, 2012 at 5:12:00 AM GMT+05:12"
string(47) "Sunday, January 1, 2012 at 5:12:00 AM GMT+05:12"
string(48) "Sunday, Tevet 6, 5772 AM at 5:12:00 AM GMT+05:12"

Warning: IntlDateFormatter::__construct(): datefmt_create: invalid value for calendar type; it must be one of IntlDateFormatter::TRADITIONAL (locale's default calendar) or IntlDateFormatter::GREGORIAN. Alternatively, it can be an IntlCalendar object in /Users/Ingwie/Work/drag0n-php/ext/intl/tests/dateformat_calendars.php on line 27
==DONE==